apps/dashboard/app/apps/ood_app.rb

```diff
@@ -184,10 +184,6 @@ def batch_connect_app?
     role == "batch_connect"
   end
 
-  def batch_connect
-    @batch_connect ||= BatchConnect::App.new(router: router)
-  end
```

Comment:
Here you can see OodApp (what is now the parent) having a reference to a `BatchConnect::App`, what is now it's child.